docs/stencil-docs/developing-further/google-analytics-enhanced-ecommerce.md

```diff
@@ -30,26 +30,25 @@ Cornerstone versions 2.6.0+ will have Data Tags already included in the theme.
 </div>
 </div>
 
-## Prerequisites
+### Prerequisites
+* [BigCommerce Store](https://support.bigcommerce.com/s/article/Starting-a-Bigcommerce-Trial)
+* [Optimized One-Page Checkout enabled](https://support.bigcommerce.com/s/article/Optimized-Single-Page-Checkout)
 
-* BigCommerce store using your store’s checkout must be using Optimized One-Page Checkout.
-
-Data Tags will work on any theme. For this tutorial, we will be adding data tags to the Cornerstone theme.
 
 ### Downloading a Theme

```

Comment:
This section should be moved to within the "Add data tags" header.
Also, since we're using Cornerstone specifically in this tutorial, it should be added to the prerequisites.